### PR TITLE
Add trigger update command

### PR DESCRIPTION
**Pull request by daisy-ycguo**
## Proposed Changes

* Add trigger update command

### CODE REVIEW

File: pkg/eventing/v1alpha1/client.go [hunk 1]
```diff
@@ -118,3 +131,70 @@ func (c *knEventingClient) Namespace() string {
...
+		triggerFilterAttributes := v1alpha1.TriggerFilterAttributes(filters)
+		b.trigger.Spec.Filter = &v1alpha1.TriggerFilter{
+			Attributes: &triggerFilterAttributes,
+		}
+	}
+	return b
+}
+
+// UpdateFilters to update the filters of trigger object
+func (b *TriggerBuilder) UpdateFilters(toUpdate map[string]string, toRemove []string) *TriggerBuilder {
```
Comment (rhuss) [inline pkg/eventing/v1alpha1/client.go]: I wouldn't do any removal or change in the builder. Its about creating an immutable object, so would prefer to have just a `Filter()` method (as in my PR ;-) if you don't mind I would update that later.

File: pkg/eventing/v1alpha1/client.go [hunk 2]
```diff
@@ -118,3 +131,70 @@ func (c *knEventingClient) Namespace() string {
...
+	}
+	for _, key := range toRemove {
+		delete(existing, key)
+	}
+	b.Filters(existing)
+	return b
+}
+
+// Sink to set the subscriber of trigger object
+func (b *TriggerBuilder) Sink(sink *duckv1.Destination) *TriggerBuilder {
```
Comment (rhuss) [inline pkg/eventing/v1alpha1/client.go]: Let's call it Subscriber like in the spec.

File: pkg/eventing/v1alpha1/client.go (hunk 1 above)
Comment (daisy-ycguo) [reply pkg/eventing/v1alpha1/client.go]: Find. I don't mind. Please update it in a separate PR.

**Pull request merged**